r_practice/r_practice.md

```diff
@@ -0,0 +1,142 @@
+
+</div>
+
+## The Data
+
+For this practice module we will be using real, publicly-available cervical cancer data from [UC Irvine's machine learning data repository](http://archive.ics.uci.edu/ml/index.php); you can download those data here: [http://archive.ics.uci.edu/ml/machine-learning-databases/00383/risk_factors_cervical_cancer.csv](http://archive.ics.uci.edu/ml/machine-learning-databases/00383/risk_factors_cervical_cancer.csv). If you have done the Python Practice Module, this dataset might be familiar! 
+
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
+
+The second link above will attempt to download a CSV file to your computer-- if you don't want to do that, or if your institution's permissions don't allow it, you can also access the dataset here: (https://archive.ics.uci.edu/ml/datasets/Cervical+cancer+%28Risk+Factors%29)[https://archive.ics.uci.edu/ml/datasets/Cervical+cancer+%28Risk+Factors%29]. If you want to download the CSV from there, just click the "Data Folder" link at the top of the page. 
+
+</div>
+
+## Practice Problem
+
+According to the UCI Machine Learning database, the Cervical Cancer (Risk Factors) dataset "focuses on the prediction of indicators/diagnosis of cervical cancer. The features cover demographic information, habits, and historic medical records." 
+ 
+Your Task
+-----
+1. Explore [the Cervical Cancer (Risk Factors) dataset](http://archive.ics.uci.edu/ml/machine-learning-databases/00383/risk_factors_cervical_cancer.csv) and 
+2. Use the R skills that you've learned to **develop a hypothesis about some likely risk factors** for a diagnosis of cervical cancer.  
+
+Remember that there are several ways to approach this task; there is no one right answer! So have fun, and good luck!
+
+### Stuck? 
+
+If this practice problem was challenging, [here is a link to an example notebook]() where we go through one possible solution to the practice problem. If you'd like to download the R Markdown notebook to work with on your own computer, you can do that in RStudio Cloud. 
+
+We encourage you to spend some time playing around with the data before you look at the example! And remember, just because your solution doesn't look exactly like the example doesn't mean it's wrong-- there are a variety of ways you might have approached the task we gave you.  
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+Feeling overwhelmed? Learning data science in R take time and practice. Don't worry if you feel like you have to go back and look and the modules over and over, or google lots of things-- folks who do this for a living google things every day! You don't have to memorize everything. Just keep your favorite resources close at hand so you remember what tools you have access to. 
```

Comment:
```suggestion
Feeling overwhelmed? Learning data science in R take time and practice. Don't worry if you feel like you have to go back and look and the modules over and over, or google lots of things-- folks who do this for a living Google things every day! You don't have to memorize everything. Just keep your favorite resources close at hand so you remember what tools you have access to. 
```
I'm not dogmatic, do what you want here

Reply:
Works for me, I have no strong feelings lol